File: Lab2KarpaBohdan36/src/Lab2KarpaBohdan36/Leaves.java
package Lab2KarpaBohdan36;

/**
 * Клас, що представляє листя рослини.
 */
public class Leaves {
    private String leafShape;

    /**
     * Конструктор, що ініціалізує форму листя.
     * @param leafShape Форма листя.
     */
    public Leaves(String leafShape) {
        this.leafShape = leafShape;
    }

    /**
     * Метод для отримання форми листя.
     * @return Форма листя.
     */
    public String getLeafShape() {
        return leafShape;
    }

    /**
     * Метод для зміни форми листя.
     * @param leafShape Нова форма листя.
     */
    public void setLeafShape(String leafShape) {
        this.leafShape = leafShape;
    }

    @Override
    public String toString() {
        return "Форма листя: " + leafShape;
    }
}

File: plants/leaves.py
class Leaves:
    """Клас, що представляє листя рослини."""

    def __init__(self, leaf_shape):
        """Ініціалізує форму листя."""
        self.leaf_shape = leaf_shape

    def get_leaf_shape(self):
        """Повертає форму листя."""
        return self.leaf_shape

    def set_leaf_shape(self, leaf_shape):
        """Задає нову форму листя."""
        self.leaf_shape = leaf_shape

    def __str__(self):
        return f"Форма листя: {self.leaf_shape}"
